video-token-backend/.owners

```diff
@@ -0,0 +1,3 @@
+anishshandilya
+# Insert your Github username here
+timmydoza
```

Comment:
@PikaJoyce can you replace both of these names and replace it with your own GitHub username?